lib/engine/game/g_18_uruguay/nationalization.rb

```diff
@@ -206,6 +221,17 @@ def event_nationalization!
           buy_train(@fce, train, :free)
         end
 
+        # Move stock price one step left for each stock more than limit
```

Comment:
Is this comment correct?

```suggestion
        # Move stock price one step left for each share more than limit
```

Reply:
No It should be loans